source/funkin/ui/debug/charting/ChartEditorState.hx

```diff
@@ -736,6 +727,21 @@ class ChartEditorState extends UIState // UIState derives from MusicBeatState
 
   // Audio
 
+  /**
+   * Play the welcome music or not.
+   */
+  var isWelcomeMusic:Bool = false;
```

Comment:
`isWelcomeMusic` doesn't communicate well what the variable does. Probably should be called `shouldPlayWelcomeMusic`.